browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -2514,18 +2507,14 @@ L.CanvasTileLayer = L.Layer.extend({
 				this._twipsToLatLng(this._cellCursorTwips.getTopLeft(), this._map.getZoom()),
 				this._twipsToLatLng(this._cellCursorTwips.getBottomRight(), this._map.getZoom()));
 
-			var start = this._twipsToCorePixels(this._cellCursorTwips.min);
-			var offsetPixels = this._twipsToCorePixels(this._cellCursorTwips.getSize());
-			this._cellCursorPixels = L.LOUtil.createRectangle(start.x, start.y, offsetPixels.x, offsetPixels.y);
-			app.file.calc.cellCursor.address = [parseInt(strTwips[4]), parseInt(strTwips[5])];
-			app.file.calc.cellCursor.rectangle.pixels = [Math.round(start.x), Math.round(start.y), Math.round(offsetPixels.x), Math.round(offsetPixels.y)];
-			app.file.calc.cellCursor.rectangle.twips = this._cellCursorTwips.toRectangle();
+			app.file.calc.cellCursor.address = new app.definitions.simplePoint(parseInt(strTwips[4]), parseInt(strTwips[5]));
+			let tempRectangle = this._cellCursorTwips.toRectangle();
+			app.file.calc.cellCursor.rectangle = new app.definitions.simpleRectangle(tempRectangle[0], tempRectangle[1], tempRectangle[2], tempRectangle[3]);
 			app.file.calc.cellCursor.visible = true;
+
 			app.sectionContainer.onCellAddressChanged();
 			if (autofillMarkerSection)
-				autofillMarkerSection.calculatePositionViaCellCursor([this._cellCursorPixels.getX2(), this._cellCursorPixels.getY2()]);
-
-			this._cellCursorXY = new L.Point(parseInt(strTwips[4]), parseInt(strTwips[5]));
+				autofillMarkerSection.calculatePositionViaCellCursor([app.file.calc.cellCursor.rectangle.pX2, app.file.calc.cellCursor.rectangle.pY2]);
```

Comment:
long names to type `app.file.calc.cellCursor.rectangle` maybe we should keep it as a member of tile layer `this.cellCursor`? (just a note to consider in later PRs for other variables, for readability)

or we can put getter here:   `cellCursor() { return app.file.calc.cellCursor; }` so it will be short

Reply:
Maybe i can replace this like below? That'll be shorter. I am also uncomfortable with the long names.

* app.file.calc.cellCursor.rectangle.
* app.calc.cellRectangle